run-geth.sh

```diff
@@ -16,8 +16,8 @@ printf "${LOG_START}Starting geth...${LOG_END}"
 geth --port 0 --networkid 1101 --identity "somerandomidentity" \
     --ws --wsaddr "127.0.0.1" --wsport "8546" --wsorigins "*" \
     --rpc --rpcport "8545" --rpcaddr "127.0.0.1" --rpccorsdomain "*" \
-    --rpcapi "db,ssh,miner,admin,eth,net,web3,personal,debug" \
-    --wsapi "db,ssh,miner,admin,eth,net,web3,personal,debug" \
+    --rpcapi "miner,admin,eth,net,web3,personal,debug" \
+    --wsapi "miner,admin,eth,net,web3,personal,debug" \
```

Comment:
Are we also changing the recommended `geth` version? Currently we recommend `1.9.9-stable` in the `manual-setup.adoc` and use the same in `deployments/local-setup-environment/provisioning/install-geth.sh` for automatic setup.

Reply:
We can (I'm running latest, and the macos-setup script installs latest anyway). I'm fine either way to be honest---this makes it work on latest and shouldn't break 1.9.9 so it also feels fine to leave it for now.

Reply:
Ok, good!